src/engine/ratecontrol.h

```diff
@@ -94,7 +125,7 @@ class RateControl : public EngineControl {
     double getTempRate(void);
 
     // Values used when temp and perm rate buttons are pressed
-    static double m_dTemp, m_dTempSmall, m_dPerm, m_dPermSmall;
+    static double m_dTemporaryRateChangeCoarse, m_dTemporaryRateChangeFine, m_dPermanentRateChangeCoarse, m_dPermanentRateChangeFine;
```

Comment:
This line is even for GitHub too long.
Please use an extra line and type for every declaration.